City-of-Helsinki/hel-fi-drupal-grants
# AU-574: Application form step 2 & draft

# [AU-574](https://helsinkisolutionoffice.atlassian.net/browse/AU-574)


## What was done


- Robot test to fill in step 2 of application form
- Robot test to save and continue on application form
- Removed extraneous scrolls

## How to test


`ROBOT_OPTIONS="-t fill_application_form" make test-robot`

Test should pass


[AU-574]: https://helsinkisolutionoffice.atlassian.net/browse/AU-574?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
